build.sbt

```diff
@@ -17,6 +19,9 @@ lazy val core = project.in(file("core"))
     tutTargetDirectory := file(".")
   )
 
+lazy val coreJS = core.js
+lazy val coreJVM = core.jvm
+
 lazy val commonSettings = Def.settings(
```

Comment:
I don't really know a lot about how the Scala.js plugin works - do the `libraryDependencies` in the `commonSettings` need to use the `%%%` separator?

Reply:
You know, im not entirely sure. As far as I can tell, `%%%` wasn't needed, but maybe there is a runtime linker problem waiting to blow up if I use it more.

Reply:
Ok, let's merge it 👍